blueprints/ember-frost-css-core/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.0.2'}
```

Comment:
In the consumer, yes. But does this repo need theme in its package.json for it to work? Meaning, the packages referenced in the addAddonsToProject, do they have to be installed? Or can it be anything in npm?